storage/mysql/log_storage.go

```diff
@@ -372,6 +373,14 @@ func (t *logTreeTX) WriteRevision(ctx context.Context) (int64, error) {
 	return t.treeTX.writeRevision, nil
 }
 
+// GetMerkleNodes returns the requested nodes at the read revision.
+func (t *logTreeTX) GetMerkleNodes(ctx context.Context, nodeIDs []tree.NodeID) ([]tree.Node, error) {
```

Comment:
This bit might look confusing: The `GetMerkleNodes` method just moved from tree storage to log storage.
Reason: tree storage (for this particular implementation) doesn't know `root.revision`, whereas the log storage does. In contrast, in other implementation (e.g. `cloudspanner` and `spanner`) tree storage does know it.

This indicates that tree vs log storage abstractions are sometimes violated / not defined. Given that we only have logs now, the bigger plan is to combine the two into one interface [#2378].